examples/auth0.rs

```diff
@@ -1,44 +1,42 @@
 "#;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let jwks: jwk::JwkSet = serde_json::from_str(JWKS_REPLY).unwrap();
+    let jwks: JwkSet = serde_json::from_str(JWKS_REPLY).unwrap();
+    let header = decode_header(TOKEN).unwrap();
 
-    let header = decode_header(TOKEN)?;
-    let kid = match header.kid {
-        Some(k) => k,
-        None => return Err("Token doesn't have a `kid` header field".into()),
+    let Some(kid) = header.kid else {
```

Comment:
Since which version of Rust is that stable? First time I'm seeing that syntax

Reply:
let-else statements is stabilized at Rust 1.65.

https://blog.rust-lang.org/2022/11/03/Rust-1.65.0.html#let-else-statements